### PR TITLE
Add JAVA_HOME for Faban

### PR DESCRIPTION
**Pull request by Xusine1131**
#295

### CODE REVIEW

**Review by neo-apz — CHANGES_REQUESTED** [review]
I believe the right way of doing this is to set the `JAVA_HOME` variable inside the Java base image so that all the other images that use this image (`from`) have this set correctly already. Look at this for example: https://github.com/parsa-epfl/cloudsuite/blob/05deab3c1909f541ded18af25733e2d68b23e712/commons/java/openjdk8/Dockerfile#L6

File: commons/java/openjdk11/Dockerfile [hunk 1]
```diff
@@ -2,3 +2,5 @@ FROM cloudsuitetest/debian:base-os
 
 RUN apt-get update -y && apt-get install -y --no-install-recommends openjdk-11-jdk-headless \
         && rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_HOME /usr/
```
Comment (neo-apz) [inline commons/java/openjdk11/Dockerfile:6]: Is this the correct path? Shouldn't it be something like `/usr/lib/jvm/...` ?

**Review by neo-apz — CHANGES_REQUESTED** [review]

File: commons/java/openjdk11/Dockerfile (hunk 1 above)
Comment (Xusine1131) [reply commons/java/openjdk11/Dockerfile:6]: It works out of box though. I think `javac` is in /usr/bin/javac now

Comment (Xusine1131) [reply commons/java/openjdk11/Dockerfile:6]: You can check the CI result.

Comment (neo-apz) [reply commons/java/openjdk11/Dockerfile:6]: This is not a good answer. You have to verify this manually by going inside the image. The check might pass due to a side effect (tests cannot prove the absence of bugs). Please double-check this. As far as I know, `JAVA_HOME` should be within that path that I wrote before.

Comment (Xusine1131) [reply commons/java/openjdk11/Dockerfile:6]: Checked. /usr/bin/javac is a link pointing to /etc/alternatives/javac, which is pointing to /usr/lib/jvm/java-11-openjdk-amd64/bin/javac.

Comment (neo-apz) [reply commons/java/openjdk11/Dockerfile:6]: So as I said `JAVA_HOME` should be `/usr/lib/jvm/java-11-openjdk-amd64`. Please note `JAVA_HOME` is not only used to access `javac`.

Comment (neo-apz) [reply commons/java/openjdk11/Dockerfile:6]: However, please note the path is different across architecture, and the above path is true only for the `amd64` image. So you need to set this properly for each image.

Comment (Xusine1131) [reply commons/java/openjdk11/Dockerfile]: Done.

**Review by neo-apz — APPROVED** [review]

**Pull request merged**